hw/top_earlgrey/syn/chip_earlgrey_asic.sdc

```diff
@@ -61,13 +61,13 @@ if {$FOUNDRY_ROOT != ""} {
     set USB_P_PIN  gen_flops[3]*.u_size_only_reg/D
     set USB_PIN    gen_flops[1]*.u_size_only_reg/D
     set USB_D_PIN  u_usb_d_flop/gen_*u_impl*/gen_flops[0]*.u_size_only_reg/Q
-    set USB_OE_PIN u_oe_flop_flop/gen_*u_impl*/gen_flops[0]*.u_size_only_reg/Q
+    set USB_OE_PIN u_oe_flop/gen_*u_impl*/gen_flops[0]*.u_size_only_reg/Q
```

Comment:
this is really odd...how did this not surface before..? did we change something recently?

Reply:
Yeah it may be my own mistake.
I introduced variables here to set these constraints about 2 weeks ago.